dist/tpl/ShellView.html

```diff
@@ -49,14 +49,14 @@ <h1 id="title"><a>Loading Library</a></h1>
 
         <div class="sidebar-header">
             <ul class="playlist-primary-tabs">
-                <li class="playlist-primary-tab xbmc-tab active" data-pane="xbmc" title="Switch to XBMC (shared) player"><i class="fa fa-rss"></i><i class="fa fa-play"></i> XBMC</li>
+                <li class="playlist-primary-tab xbmc-tab active" data-pane="xbmc" title="Switch to Kodi (shared) player"><i class="fa fa-rss"></i><i class="fa fa-play"></i> XBMC</li>
                 <li class="playlist-primary-tab local-tab" data-pane="local" title="Switch to playback in your browser"><i class="fa fa-headphones"></i><i class="fa fa-play"></i> Local</li>
             </ul>
             <div class="playlist-actions-wrapper"></div>
         </div>
 
         <div class="sidebar-second-content">
-            <div class="sidebar-pane" id="playlist-xbmc"></div>
```

Comment:
Happy trigger finger, It will be reverted in one minute.

One thing, Ide like to ask you while Im here, would you like to fix the bad usages of camel case in some titles while we at it?